go/consensus/tendermint/abci/mux.go

```diff
@@ -60,13 +60,15 @@ var (
 )
 
 // ApplicationConfig is the configuration for the consensus application.
-type ApplicationConfig struct {
-	DataDir         string
-	StorageBackend  string
-	Pruning         PruneConfig
-	HaltEpochHeight epochtime.EpochTime
-	MinGasPrice     uint64
-	DisableCheckTx  bool
+type ApplicationConfig struct { // nolint: maligned
+	DataDir             string
+	StorageBackend      string
+	Pruning             PruneConfig
+	HaltEpochHeight     epochtime.EpochTime
+	MinGasPrice         uint64
+	DisableCheckTx      bool
+	DisableCheckpointer bool
+	CheckpointInterval  time.Duration
```

Comment:
Could you move the checkpointer options under its own group (e.g., add empty line above the first option).